README.Rmd

```diff
@@ -24,13 +23,26 @@ knitr::opts_chunk$set(
 )
 ```
 
-Create purls from the filtered sands of your dependencies, powered by [OSS Index](https://ossindex.sonatype.org/)
+__Note__, this README is based on the development version, which is significantly 
+different to the CRAN (v0.0.3) version.
+
+## Overview
+
+This package aims to secure your R package against insecure dependencies using the [OSS Index](https://ossindex.sonatype.org/). Essentially, we check R packages for
```

Comment:
Maybe instead of "secure your R package", "secure your R projects"? Since I think the aim is for this tool to be useful to all kinds of folks using R, not just package maintainers.

Reply:
Changed.